README.md

```diff
@@ -127,3 +127,153 @@ To play with newly trained agents, just follow the instructions in the [Overcook
 All DRL results can be reproduced by running the `.sh` scripts under `human_aware_rl/experiments/`.
 
 All non-DRL results can be reproduced by running cells in `NeurIPS Experiments and Visualizations.ipynb`.
+
+# Rllib
+
+Some of the agents are now compatible to be trained with rllib, using Tensorflow 2. This requires alternative installation. Make sure you are on the branch `rllib`
+
+## Installation
+
+Ensure that have the correct `overcooked-ai` submodule code (if you cloned directly into the `rllib` branch this should be done automatically)
+
+```bash
+$ cd overcooked_ai
+overcooked_ai $ git checkout overcooked_ai_improvements
+overcooked_ai $ cd ..
+```
+
+Now create a new conda environment and run the install script as before
+
+```bash
+$ conda create -n harl_rllib python=3.7
+$ conda activate harl_rllib
+(harl_rllib) $ cd human_ai_coord
+(harl_rllib) human_ai_coord $ ./install.sh
+```
+
+Finally, install the latest stable version of tensorflow compatible with rllib
+```bash
+(harl_rllib) $ pip install tensorflow==2.1.0
+```
+Or, if working on the BAIR servers, install a version of tensorflow and cuDNN that is compatible with the available nvidia drivers to use GPUs
+```bash
+(harl_rllib) $ pip install tensorflow-gpu==2.0.0
+(harl_rllib) $ conda install -c anaconda cudnn=7.6.0
+```
+
+
+The behavior cloning model (and reproducibility tests) requires data from a different git repo. Thus, it must be installed from source.
+From outside the `human_ai_coord` repo:
+```bash
+(harl_rllib) $ git clone https://github.com/HumanCompatibleAI/human_ai_MSR.git
+(harl_rllib) $ cd human_ai_MSR
+human_ai_MSR $ git checkout nathan_dev
+(harl_rllib) $ human_ai_MSR $ pip install -e .
+```
+
+Your virtual environment should now be configured to run the rllib training code. Verify it by running the following command 
+
+```bash
+python -c "from ray import rllib"
+```
+If no errors are reported, you're good to go! If you get the error `ModuleNotFoundError: No module named 'google.protobuf'`, run `conda install protobuf`. Note: you MUST use conda for this fix to work; trying to use pip here doesn't fix the issue.
```

Comment:
I added this at the end because it seemed to be a non-deterministic error I encountered. Added `conda install protobuf` to the install script and was able to install everything just fine, so hopefully that prevents the issue from cropping up at all